blueocean-config/src/main/java/io/jenkins/blueocean/config/BlueOceanConfig.java

```diff
@@ -0,0 +1,22 @@
+package io.jenkins.blueocean.config;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Holds the Blue Ocean configuration
+ *
+ */
+public class BlueOceanConfig {
+    /**
+     * Indicates if the Organizations feature should be enabled
+     */
+    public Boolean organizationEnabled;
```

Comment:
should be a private final with a getter to be consistent with other model objects.